### ISSUE TITLE
API BluePrint Version?

### ISSUE THREAD

**Issue by export-mike** [TDispatch/Passenger-API#8]
Hey I tell what would be very helpful, is an API blueprint version of the API, one which we devs can host on Apiary and develop against.

https://apiblueprint.org/
https://apiary.io/

Cheers

**Comment by varunachar** [comment]
Agreed. They're doc also is outdated vs they're actual API. Auto generation tools are essential these days.. Or simply create a [postman ](getpostman.com)collection link!

Adding Swagger to the list as well. It has a pretty decent tooling system as well.
